code/modules/surgery/organs/organ_external.dm

```diff
@@ -268,9 +274,9 @@
 	//If limb took enough damage, try to cut or tear it off
 	if(owner)
 		if(sharp && !(limb_flags & CANNOT_DISMEMBER))
-			if(brute_dam >= max_damage && prob(brute))
+			if(limb_brute >= max_damage && prob(original_brute / 2))
 				droplimb(0, DROPLIMB_SHARP)
-			if(burn_dam >= max_damage && prob(burn))
+			if(limb_burn >= max_damage && prob(original_burn / 2))
 				droplimb(0, DROPLIMB_BURN)
```

Comment:
why not just call this before the damage is done?

Reply:
any more on this @Qwertytoforty

Reply:
Sorry, was on vacation but replied on github, worried about it breaking by being delimbed too early, and preserving current functionality of when in the tree the limb breaks off.